internal/cli/command/config/config.go

```diff
@@ -53,26 +53,26 @@ func NewCmd(fileUtil file.File) *cobra.Command {
 			} else {
 				err := fileUtil.CreateDirIfNotExist(config.ConfigFileDir())
 				if err != nil {
-					log.Error(err, "error in creating config file directory")
+					log.Error(err, "error in creating kubeconfig file directory")
 					return
 				}
 				err = fileUtil.CreateFile(configFilePath)
 				if err != nil {
-					log.Error(err, "error in creating config file")
+					log.Error(err, "error in creating kubeconfig file")
 					return
 				}
 			}
 
 			var configFileContent string
 			configFileContent += fmt.Sprintf("%s: %s\n", config.OctaviusCPHost, cpHost)
-			//TODO: config.EmailId should be config.EmailID
+			//TODO: kubeconfig.EmailId should be kubeconfig.EmailID
```

Comment:
can you please resolve this TODO and replace EmailId to EmailID in all the configs needed